<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QMainWindow" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qChess</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="newGame">
         <property name="text">
          <string>Новая игра</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="options">
         <property name="text">
          <string>Настройки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetOpt" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Белый игрок:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Тип игрока</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="humen1">
              <property name="text">
               <string>Человек</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ai1">
              <property name="text">
               <string>Компьютер</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="cofW1" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_kof1">
               <property name="text">
                <string>Сложность</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="kof1">
               <item>
                <property name="text">
                 <string>Легко</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Средне</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Тяжело</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Черный игрок:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Тип игрока</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="humen2">
              <property name="text">
               <string>Человек</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ai2">
              <property name="text">
               <string>Компьютер</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="cofW2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_kof2">
               <property name="text">
                <string>Сложность</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="kof2">
               <item>
                <property name="text">
                 <string>Легко</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Средне</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Тяжело</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="save">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cansle">
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
